source/index.html.md

```diff
@@ -29,19 +29,19 @@ GOV.UK.
 
 To get started:
 
- 1. Pick a page on GOV.UK eg. [https://www.gov.uk/take-pet-abroad](https://www.gov.uk/take-pet-abroad)
- 2. Make a note of the path eg. `/take-pet-abroad`
+ 1. Pick a page on GOV.UK, for example: [https://www.gov.uk/take-pet-abroad](https://www.gov.uk/take-pet-abroad)
+ 2. Make a note of the path, for example: `/take-pet-abroad`
  3. Using a tool such as [curl](https://curl.haxx.se/),
     [Postman](https://www.getpostman.com/) or your web browser make a GET
     request to [`https://www.gov.uk/api/content/take-pet-abroad`](https://www.gov.uk/api/content/take-pet-abroad)
  4. You’ll receive a JSON response, the fields for this are
     explained in the [reference documentation](reference.html)
 
-For example, using [curl](https://curl.haxx.se/) command line utility and the
-[jq](https://stedolan.github.io/jq/) tool to display the response:
+For example, using [curl](https://curl.haxx.se/) command line utility tool to
```

Comment:
I'd probably cut the display the response part of this now we don't have jq (which was used to display the response)